### ISSUE TITLE
Cannot build pyobjc-framework-PubSub on macOS 10.15

### ISSUE THREAD

**Issue by buchi** [ronaldoussoren/pyobjc#299]
When trying to build and install pyobjc from source on macOS 10.15 with
```
pip install --no-binary :all: pyobjc==6.2
```
it fails with the following error:
```
error: option --single-version-externally-managed not recognized
```

[pyobjc-framework-PubSub/setup.py](https://github.com/ronaldoussoren/pyobjc/blob/v6.2/pyobjc-framework-PubSub/setup.py#L21) declares `max_os_level="10.14"`. However [pyobjc/setup.py](https://github.com/ronaldoussoren/pyobjc/blob/v6.2/pyobjc/setup.py#L112) doesn't list the framework as removed in 10.15. I guess pyobjc/setup.py should mark it as removed in 10.15 as well to avoid trying to install it.

**Comment by ronaldoussoren** [comment]
Good catch!

I've fixed this in changeset ef0010cb3c5c6ee0c99e23035712edab3ec73f5b.

**Comment by buchi** [comment]
Thanks!
